tldw_Server_API/app/core/LLM_Calls/providers/mlx_provider.py

```
@@ -0,0 +1,568 @@
+        temp = request.get("temperature") or request.get("temp")
+        if temp is not None:
+            out["temp"] = temp
+        top_p = request.get("top_p") or request.get("topp")
+        if top_p is not None:
+            out["top_p"] = top_p
+        top_k = request.get("top_k") or request.get("topk")
+        if top_k is not None:
+            out["top_k"] = top_k
+        return out
+
+    def chat(self, request: Dict[str, Any], *, timeout: Optional[float] = None) -> Dict[str, Any]:
```

Ruff (0.14.6)

Unused method argument: `timeout`

(ARG002)